e2e/cypress/tests/support/email.ts

```diff
@@ -26,3 +29,23 @@ Cypress.Commands.add('getRecentEmail', ({username, email}) => {
         return cy.wrap({...data, body});
     });
 });
+
+declare global {
+    namespace Cypress {
+        interface Chainable {
+
+        /**
+         * getRecentEmail is a task to get an email sent to a user
+         * from the email service provider
+         * @param options.username - username of the user
+         * @param options.email - email of the user
+         *
+         * @example
+         *   cy.getRecentEmail().then((data) => {
+         *       // do something with the email data/content
+         *   });
+         */
+        getRecentEmail(options: {user: Pick<UserProfile, 'username' | 'email'>}): Chainable;
```

Comment:
This should be `getRecentEmail(options: Pick<UserProfile, 'username' | 'email'>): Chainable;` to match existing usages.